### PR TITLE
Remove default user

### PR DESCRIPTION
**Pull request by recca0120**

### CODE REVIEW

File: src/app/core/authentication/login.service.ts [hunk 1]
```diff
@@ -29,4 +35,8 @@ export class LoginService {
   menu() {
     return this.http.get<{ menu: Menu[] }>('/me/menu').pipe(map(res => res.menu));
   }
+
+  setDefaults(user: User) {
+    return Object.assign(this.defaults, user);
+  }
 }
```
Comment (nzbin) [inline src/app/core/authentication/login.service.ts:42]: I think it‘s more convenient to set default user in `user.ts` and the `login.service.ts` should only have APIs.

Comment (recca0120) [reply src/app/core/authentication/login.service.ts]: I  add Config and ConfigService, the developer can use AUTH_CONFIG to set API Url and user default value

**Comment by nzbin** [comment]
Many thanks for this PR, but I won't merge it because it's against my original design intent. In addition, I think the `login.service.ts` is a better choice to be the configuration.

**Comment by recca0120** [comment]
> 

if it can keep login service at the same time?

**Comment by nzbin** [comment]
In fact, there has a `settings.ts` to be customized some config. The project is a template and the auth module maybe unnecessary. I hope the project can be easy to use for some junior developer who unfamiliar with Angular.

**Comment by recca0120** [comment]
ok. it's fine

**Pull request closed without merging**